### ISSUE TITLE
14.2 lab fails because we're on CentOS but it's not included in the params.pp file

### ISSUE THREAD

**Issue by 16c7x** [puppetlabs/ilt-practitioner-issues#43]
## Describe the Bug

Facter returns the following
[root@2262nix0 serverspec]# facter operatingsystem
CentOS

But the apache/manifests/params.pp file only has options for redhat or debian. To rectify the problem line 3 of params has to be changed from redhat to CentOS.

## Expected Behavior
Running Puppet apply should apply the apache class successfully.

## Steps to Reproduce
run puppet apply ../examples/init.pp --modulepath ~/development
from within the specserver directory.

## Environment
 - Version [e.g. 1.27.0]
 - Platform [e.g. Ubuntu 18.04]

## Additional Context
```
[root@2262nix0 serverspec]# puppet apply ../examples/init.pp --modulepath /root/development
Warning: Unknown variable: 'apache::params::package'. (file: /root/development/apache/manifests/init.pp, line: 40, column: 15)
Warning: Unknown variable: 'apache::params::config'. (file: /root/development/apache/manifests/init.pp, line: 44, column: 16)
Warning: Unknown variable: 'apache::params::service'. (file: /root/development/apache/manifests/init.pp, line: 50, column: 18)
Notice: Compiled catalog for 2262nix0.classroom.puppet.com in environment production in 0.02 seconds
Error: Parameter path failed on File[apache_config]: File paths must be fully qualified, not 'apache_config' (file: /root/development/apache/manifests/init.pp, line: 43)
```

**Comment by tuxmea** [comment]
This module is used for introducing testing of Puppet Code.
People should recognize the error and fix the code.

**Comment by greglarkin** [comment]
Yes, the point of this lab is that the code contains errors, and the intent is that the code and/or tests should be fixed in order to get a clean test run. That will require the student to identify Puppet coding errors as written and correct them.

**Comment by jessereynolds** [comment]
This is a great demonstration of why we need the instructor notes (on each slide for the presenter) - to convey the point of the slide / lab.
